src/posix/platform/settings.cpp

```diff
@@ -167,89 +166,17 @@ static void swapDiscard(otInstance *aInstance, int aFd)
     VerifyOrDie(0 == unlink(swapFileName), OT_EXIT_ERROR_ERRNO);
 }
 
-void otPlatSettingsInit(otInstance *aInstance)
```

Comment:
Looks like these functions are moved to the bottom of the file and otherwise seem to be practically mostly the same code.

Should we keep them same place to make the `diff` simpler to read?

Reply:
I moved these API definitions. There are still some moving codes in diff, but should be better now.